packages/link/Link.js

```diff
@@ -3,32 +3,54 @@ import PropTypes from 'prop-types';
 import { isAbsoluteUrl } from '@availity/resolve-url';
 
 // if absolute or loadApp is disabled, return url. otherwise loadappify the url
-export const getUrl = (url = '', loadApp) => {
-  const absolute = isAbsoluteUrl(url);
+export const getUrl = (url = '', loadApp, absolute) => {
   if (absolute || !loadApp) return url;
 
   return `/public/apps/home/#!/loadApp?appUrl=${encodeURIComponent(url)}`;
 };
 
+// takes href and transforms it so that we can compare hostnames and other properties
+const getLocation = href => {
+  const location = document.createElement('a');
+  location.href = href;
+  return location;
+};
+
 const AvLink = ({
   tag: Tag,
-  href: url,
+  href,
   target,
   children,
   onClick,
   loadApp,
+  rel,
   ...props
-}) => (
-  <Tag
-    href={getUrl(url, loadApp)}
-    target={target}
-    onClick={event => onClick && onClick(event, getUrl(url, loadApp))}
-    data-testid="av-link-tag"
-    {...props}
-  >
-    {children}
-  </Tag>
-);
+}) => {
+  const absolute = isAbsoluteUrl(href);
+  const url = getUrl(href, loadApp, absolute);
+
+  // eslint-disable-next-line no-cond-assign
+  if ((target = '_blank') && !rel && absolute) {
```

Comment:
https://eslint.org/docs/rules/no-cond-assign has this to say:

> There are valid reasons to use assignment operators in conditional statements.

I think we should be explicitly checking if `target` is equal to `"_blank"`
```
if (target === '_blank' && ...